servers/tests/org.fusesource.ide.server.karaf.core.tests/src/test/java/org/fusesource/ide/server/karaf/core/server/subsystems/KarafUtilsTest.java

```diff
@@ -37,6 +39,13 @@
 	public TemporaryFolder testFolder = new TemporaryFolder();
 	
 	@Test
+	public void testGetBundleSymbolicNameWithoutParameters() throws CoreException {
```

Comment:
ok, it depends to what we report the "withoutParameters" I had in mind that it was the case we are testing and not the method name
with the other tests, it will be clearer :-)